src/makeInfinityAdder.js

```diff
@@ -5,8 +5,24 @@
  * @return {function}
  */
 
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(firstValue = 0) {
+  let sumOfOperations = firstValue;
+
+  const adder = (lastValue) => {
```

Comment:
необязательно передавать в параметры firstValue со значением 0
можно просто присвоить sumOfOperations 0 и в adder передавать просто value
```suggestion
function makeInfinityAdder() {
  let sumOfOperations = 0;

  const adder = (value) => {
```